spec.md

```diff
@@ -398,6 +398,34 @@ Example error response (invalid `id` provided):
 
 ### Resource Data Model
 
+A resource is a named (addressable) blob stored in a Space, with metadata.
+The data model is derived from [W3C FileAPI: File 
+Interface](https://w3c.github.io/FileAPI/#file-section), but with the addition
+of a few crucial properties.
+
+In similar storage systems, a resource is called "File", "Object", "Document",
+"Row", "Graph", and so on.
+
+Resource properties:
+
+* `id`
+* `contents` (Blob)
+* `name` - optional
+* (tbd) a link to a **system-meta** auxiliary object (determined by the local
+  system)
+* (tbd) one or more links to **custom-meta** auxiliary objects (unlike
+  system-meta, these are modifiable by the object's controller)
+
+Example **system-meta** properties (not directly modifiable, controlled by the
```

Comment:
there are a lot of ways we could communicate this information that don't necessarily involve system-meta properties. I would prefer we not speculate on examples here that we haven't implemented because it could cause confusion if/when we accomodate the implicit use cases here in other ways that do not involve a separate auxilary object or notion of system-meta.

To be honest, I think we should not use the term 'system-meta' at all unless/until there is a whole section explaining it, and that it should be a separate PR from this one, which I think is 90% only about the resource data model orthogonal to notions of meta.

Reply:
Good point, removing it for now